packages/commonwealth/package.json

```diff
@@ -30,8 +30,7 @@
     "start-dbNode": "WORKER_NUMBER=0 NUM_WORKERS=1 ts-node --project server/tsconfig.json server/scripts/dbNode.ts",
     "start-prerender": "ts-node --project tsconfig.node.json server/scripts/runPrerenderService.ts",
     "send-notification-digest-emails": "SEND_EMAILS=true ts-node --project tsconfig.node.json server.ts",
-    "migrate-events": "ENTITY_MIGRATION=all ts-node --project tsconfig.node.json server/scripts/migrateChainEntities.ts",
-    "migrate-event": "ts-node --project tsconfig.node.json server/scripts/migrateChainEntities.ts",
```

Comment:
We use migrateEvent often -- please do not remove it. This is how we onboard a new chain (e.g. if chain A onboards, we run `ENTITY_MIGRATION=A yarn migrate-event` to populate prior events.)

Reply:
I didn't remove it. I changed the functionality so that we don't have 2 separate commands for the same thing. Currently, if you do `yarn migrate-events` it will migrate all event for all supported chains and `yarn migrate-event` needs `ENTITY_MIGRATION` to be defined to run properly. I refactored it to be a little clearer. Now there is a single `yarn migrate-events` command. If you run just the command it will migrate for all chains much like if you used ENTITY_MIGRATION='all' but if you want to migrate for just a single chain simply provide the `CHAIN_ID` for the same command. Having 2 commands that do the same thing was just redundant.